gnuradio-runtime/include/gnuradio/top_block.h

```diff
@@ -13,6 +13,7 @@
 
 #include <gnuradio/api.h>
 #include <gnuradio/hier_block2.h>
+#include <gnuradio/thread/basic_error_handler.h>
```

Comment:
This requires updating the bindings for this file.

Reply:
hi
it's a long time.
can you help me with update bindings?
I created a new PR https://github.com/gnuradio/gnuradio/pull/4795